src/components/bcros/alert/index.vue

```diff
@@ -99,4 +99,13 @@ const contact = computed((): string => {
 
   return t('alerts.contact')
 })
+
+const bcrosContacts = computed(() => {
+  const contacts = getContactInfo('registries')
+  if (AlertTypesE.FROZEN === props.alert.alertType) {
+    return [contacts[2]]
```

Comment:
It would be nice to have an enum here (even though I see the legend on lines 86-89).

Also, why do you need this computed when  you already have a computed on line 85 that seems to do something very similar?

Reply:
I was thinking of moving all under the watch, and doing everything there, but went against it. As the computed are less cpu intensive and stuff is more clearly separated for each display item.

we have two items, calculated on the same thing but with different logic.
One, the contacts list, changes only when business is frozen (needs to display only email) 
Other, the text, changes more for different types of alerts, and is not affected by frozen state for now